src/plugins/intel_gpu/src/plugin/ops/convert.cpp

```diff
@@ -23,9 +23,7 @@ static void CreateConvertLikeOp(Program& p, const std::shared_ptr<ngraph::op::v1
     auto reorderPrim = cldnn::reorder(layerName,
                                       inputs[0],
                                       cldnn::format::any,
-                                      outDataType,
-                                      std::vector<float>(),
-                                      cldnn::reorder_mean_mode::subtract);
+                                      outDataType);
```

Comment:
Could you let me know which test required this change?

Reply:
None, it's just the default parameters remaining after removing one of the arguments. This change does not affect the behavior